src/openzaak/components/documenten/tests/test_lock_eio.py

```diff
@@ -142,9 +141,9 @@ def test_unlock_success(self):
 
         self.assertEqual(eio.lock, '')
 
+    @skip('Current implementation is without authentication')
```

Comment:
I actually have no idea what's common here - for me the _feeling_ is the reverse.

I mostly find it confusing because a lot of ZTC tests are skipped because functionality is not part of the spec yet, and it's hard to tell the difference with these skipped tests. IMO it's easier to hunt down the `expectedFailure`s instead of `skip`s